// Java program to demonstrate exception handling.
class ThrowsExecp{
	public static void main(String args[]){
	        try{
                           String str = null;
		System.out.println(str.length());
                           }
                      catch(NullPointerException e){
                            System.out.println(e);
                           }
                            System.out.println(“rest of the code”);//rest of the code
	}
}
